docs/platforms/javascript/guides/remix/manual-setup.mdx

```diff
@@ -209,24 +237,18 @@ To enable readable stack traces, <PlatformLink to="/sourcemaps">configure source
 
 ## Custom Express Server
 
-If you use a custom Express server in your Remix application, you should wrap your [`createRequestHandler` function](https://remix.run/docs/en/main/other-api/adapter#createrequesthandler) manually with `wrapExpressCreateRequestHandler`. This isn't necessary if you're using the built-in Remix App Server.
+You can import your server instrumentation file at the top of your Express server implementation.
 
-<Note>
-
-`wrapExpressCreateRequestHandler` is available starting with version 7.11.0.
-
-</Note>
-
-```typescript {filename: server/index.ts}
-import { wrapExpressCreateRequestHandler } from "@sentry/remix";
-import { createRequestHandler } from "@remix-run/express";
+```typescript {filename: server.ts}
+// import the Sentry instrumentation file before anything else.
+import './instrument.server.mjs';
+// alternatively `require('./instrument.server.cjs')`
 
 // ...
 
-const createSentryRequestHandler =
-  wrapExpressCreateRequestHandler(createRequestHandler);
+const app = express();
 
-app.all("*", createSentryRequestHandler(/* ... */));
+// ...
 ```
 
 ### Usage with Vite development mode (only for SDK versions < 7.106.0)
```

Comment:
Do these docs still apply?

Reply:
That version doesn't feel very relevant anymore. Should I delete the section?

Reply:
Let's remove it!